src/components/panes/ImportLogPane.scss

```diff
@@ -0,0 +1,88 @@
+.ImportLogPane {
+    .InfoListItem-status {
+        &:before {
+            @include icon($fa-var-question-circle);
+        }
+    }
+
+    .InfoListItem-accepted {
+        &:before {
+            @include icon($fa-var-question-circle);
+        }
+    }
+
+    .InfoListItem-completed {
+        &:before {
+            @include icon($fa-var-check-circle);
+        }
+    }
+
+    .InfoListItem-error {
+        &:before {
+            @include icon($fa-var-exclamation-triangle);
+        }
+    }
+
+    .InfoListItem-imported {
+        &:before {
+            @include icon($fa-var-cloud-upload);
```

Comment:
This is actually (with some acceptable exceptions) the number of rows in the file, so I think the `fa-var-list` icon should work well here.